pages/linux/qm-help.md

```diff
@@ -0,0 +1,12 @@
+# qm help
+
+> Display help for a specific command.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Display help for a specific command:
+
+`qm help {{command}}`
+
+- Display help for a specific command with detailed information:
+
+`qm help {{command}} --verbose {{boolean}}`
```

Comment:
For this example we list the 2 possible values. Works with `1|0` or `true|false`?
```suggestion
`qm help {{command}} --verbose {{true|false}}`
```